.tekton/client-server-cg-pull-request.yaml

```diff
@@ -340,6 +327,8 @@ spec:
         value: $(tasks.build-container.results.IMAGE_DIGEST)
       - name: image-url
         value: $(tasks.build-container.results.IMAGE_URL)
+      - name: SOURCE_ARTIFACT
```

Comment:
This Param is in the wron place it should be part of the `sast-snyk-check-oci-ta` task eg https://github.com/securesign/gitsign/blob/main/.tekton/gitsign-pull-request.yaml#L298

Best to check the other one's too :)